_haikus/dns-haiku.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: DNS
+author: geekthattweaks
+---
+
+Dynamic naming<br>
+Causes the most problems for me<br>
```

Comment:
This line needs one fewer syllable

Reply:
Ahhh, my bad. Pushed a fix. Thanks for the feedback